Dockerfile.release

```diff
@@ -3,20 +3,22 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3
 LABEL name="MinIO" \
       vendor="MinIO Inc <dev@min.io>" \
       maintainer="MinIO Inc <dev@min.io>" \
-      version="v0.12.1" \
-      release="v0.12.1" \
+      version="v0.13.3" \
+      release="v0.13.3" \
       summary="KES is a stateless and distributed key-management system for high-performance applications." \
       description="KES as the bridge between modern applications - running as containers on Kubernetes - and centralized KMS solutions. Therefore, KES has been designed to be simple, scalable and secure by default. It has just a few knobs to tweak instead of a complex configuration and does not require a deep understanding of secure key-management or cryptography."
 
+ENV VERSION=v0.13.3
+
 RUN \
     microdnf update --nodocs && \
     microdnf install ca-certificates --nodocs && \
     microdnf clean all && \
     mkdir /licenses && \
     curl -s -q https://raw.githubusercontent.com/minio/kes/master/CREDITS -o /licenses/CREDITS && \
-    curl -s -q https://raw.githubusercontent.com/minio/kes/master/LICENSE -o /licenses/LICENSE
-
-COPY kes /kes
+    curl -s -q https://raw.githubusercontent.com/minio/kes/master/LICENSE -o /licenses/LICENSE && \
+    curl -sSL -q https://github.com/minio/kes/releases/download/$VERSION/kes-linux-amd64 -o kes && \
```

Comment:
Well the build is `goreleaser` based so binary is not yet uploaded to Github before container is built @aead so `COPY kes /kes` is correct

Reply:
But the binary seemed to be from master inside the released container image, ref #100 . Some issue in goreleaser?

Reply:
I would prefer a image that contains the binary. My (ideal) expectation would be:
 - The image is self-contained and just contains the absolute minimal dependencies to start the binary. Ideally a scratch container but it seems like RH doesn't provide such a UBI base.
 - The KES binary version matches the image tag. So, when I start a container with the image `v0.13.0` the KES binary should be the release `v0.13.0` - not latest or anything else.

Currently, the binary that is included is is not the release that we create. 

***

A similar problem exists in the MinIO docker file. There we download the latest release - regardless what the container image tag is. See: https://github.com/minio/minio/blob/master/Dockerfile.release#L31
That has two problems:
 - The UBI LABEL https://github.com/minio/minio/blob/master/Dockerfile.release#L8 may not reflect what is actually deployed.
 - The binary that is run, has nothing to do with the release tagged by the image.

Reply:
The binary uploaded is the binary built locally @aead - we will fix all of that properly the container is not fixed properly in this PR since you don't have access to the release mechanisms.

Reply:
> A similar problem exists in the MinIO docker file. There we download the latest release - regardless what the container image tag is. See: https://github.com/minio/minio/blob/master/Dockerfile.release#L31

MinIO's problem is different release style is different not the same problem.

Reply:
No one really should be building their own containers that are released - even in case of MinIO it is always built during release.